client.go

```diff
@@ -275,13 +274,25 @@ func (client *Client) MultiGet(filters ...string) (ranges []Range,
 	if len(filters) == 0 {
 		return
 	}
-	args := []interface{}{"FILTER"}
-	for _, filter := range filters {
-		args = append(args, filter)
+	args := createMultiGetCmdArguments(DefaultMultiGetOptions, filters)
+	reply, err = conn.Do("TS.MGET", args...)
+	if err != nil {
+		return
 	}
+	ranges, err = ParseRangesSingleDataPoint(reply)
+	return
+}
 
+// MultiGetWithOptions - Get the last samples matching the specific filters.
+// args:
+// multiGetOptions - MultiGetOptions options. You can use the default DefaultMultiGetOptions
+// filters - list of filters e.g. "a=bb", "b!=aa"
+func (client *Client) MultiGetWithOptions(multiGetOptions MultiGetOptions, filters ...string) (ranges []Range, err error) {
```

Comment:
@filipecosta90 why not merged both of this functions?
You can make `multiGetOptions` a pointer and check if its `nil` if it is, use the default options otherwise use the giving options.

Reply:
agree :) doing the changes :+1:

Reply:
solved on the latest commit @danni-m

Reply:
> @filipecosta90 Im think its pretty good, I would reconsider having 2 functions.

@danni-m I believe that for being backwards compatible we need them both. Notwithstanding followed your advise and merge the internals of both. wdyt?